atc/worker/worker.go

```diff
@@ -627,7 +658,7 @@ func (worker *gardenWorker) cloneRemoteVolumes(
 			},
 			container,
 			teamID,
-			nonLocalInput.desiredMountPath,
+			"",
```

Comment:
Does this work when there are multiple inputs that need to be streamed? `FindOrCreateVolumeForContainer` calls `db.VolumeRepository.FindContainerVolume` which looks for a volume on the worker with `c.container_id == container.ID()` and `c.path == mountPath` - but if we set the mount path to `""` for each non-local volume, if we have 2 remote inputs, won't  `FindOrCreateVolumeForContainer` think that the volume already exists and just create 1 volume?

Reply:
If my thinking is correct, then *maybe* this could be hacked around by prefixing the `desiredMountPath` with some string that couldn't form a valid path (e.g. `/tmp/build/foo` -> `streamed:/tmp/build/foo`). But this isn't the most satisfying solution 😁

Reply:
@aoldershaw Good catch! And a big issue. I missed to test multiple task inputs.